test/CloneFactory.test.js

```diff
@@ -0,0 +1,46 @@
+const SimpleToken = artifacts.require('SimpleToken');
+const SimpleCloneFactory = artifacts.require('SimpleCloneFactory');
+
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('CloneFactory', function ([_, owner]) {
+  let factory;
+
+  describe('factory initialized', function () {
+    beforeEach(async function () {
+      factory = await initFactory();
```

Comment:
Since https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1380 we've stored this kind of stuff in `this`, it'd be great for consistency if we kept doing that :)